specification/cognitiveservices/OpenAI.Inference/models/completions/azure_chat_extensions.tsp

```diff
@@ -67,5 +88,86 @@ model AzureChatExtensionsMessageContext {
     course of generating a chat completions response that was augmented by capabilities from Azure OpenAI chat
     extensions.
   """)
-  messages?: ChatMessage[];
+  messages?: ChatResponseMessage[];
+}
+
+@added(ServiceApiVersions.v2023_12_01_Preview)
+@doc("A representation of the available options for the Azure OpenAI grounding enhancement.")
+model AzureChatGroundingEnhancementConfiguration {
+  @doc("Specifies whether the enhancement is enabled.")
```

Comment:
The documentation in the `yaml` is the following: `Request object to specify if grounding enhancement is needed.`. I can't quite put my finger on it, but it sounds like there is some nuance lost (or added) with the different wording. If possible, would be cool to document briefly what is "grounding" in this context.

Reply:
Enhancements needs a major pass in general. The naming doesn't seem to reflect the members, the documentation doesn't seem to reflect the data, etc.. We'll want to follow up on that in depth and add whatever detail we can to the docs as we test, exercise, and support it.